javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -113,197 +113,55 @@
-                  },
-                  "firefox_android": {
-                    "version_added": "90"
-                  },
-                  "ie": {
-                    "version_added": false
-                  },
-                  "nodejs": {
-                    "version_added": false
-                  },
-                  "opera": {
-                    "version_added": false
-                  },
-                  "opera_android": {
-                    "version_added": false
-                  },
-                  "safari": {
-                    "version_added": "14.1"
-                  },
-                  "safari_ios": {
-                    "version_added": "14.5"
-                  },
-                  "samsunginternet_android": {
-                    "version_added": false
-                  },
-                  "webview_android": {
-                    "version_added": "92"
-                  }
-                },
-                "status": {
-                  "experimental": false,
-                  "standard_track": true,
-                  "deprecated": false
-                }
-              }
-            },
-            "fractionalSecondDigits": {
-              "__compat": {
-                "support": {
-                  "chrome": {
-                    "version_added": "84"
-                  },
-                  "chrome_android": {
-                    "version_added": "84"
-                  },
-                  "edge": {
-                    "version_added": "84"
-                  },
-                  "firefox": {
-                    "version_added": "84"
-                  },
-                  "firefox_android": {
-                    "version_added": "84"
-                  },
-                  "ie": {
-                    "version_added": false
-                  },
-                  "nodejs": {
-                    "version_added": "14.6.0"
-                  },
-                  "opera": {
-                    "version_added": "70"
-                  },
-                  "opera_android": {
-                    "version_added": "60"
-                  },
-                  "safari": {
-                    "version_added": "14.1"
-                  },
-                  "safari_ios": {
-                    "version_added": "14.5"
-                  },
-                  "samsunginternet_android": {
-                    "version_added": "14.0"
-                  },
-                  "webview_android": {
-                    "version_added": "84"
-                  }
-                },
-                "status": {
-                  "experimental": false,
-                  "standard_track": true,
-                  "deprecated": false
-                }
-              }
-            },
-            "hourCycle": {
-              "__compat": {
-                "support": {
-                  "chrome": {
-                    "version_added": "73"
+                    "version_added": "24"
```

Comment:
Note, this just copies the parent compat info - assumption is that this was added in first version